### ISSUE TITLE
[fvt]2.12.1xcatprobe image -c need to modify if no node 

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1403]
env:ubuntu
build:latest build
How to reproduce:
```root@c910f03c05k05:/opt/xcat/bin# xcatprobe image -c
Smartmatch is experimental at /opt/xcat/bin/xcatprobe line 64.
Smartmatch is experimental at /opt/xcat/bin/xcatprobe line 173.
Use of uninitialized value $node_image_table in concatenation (.) or string at /opt/xcat/probe//subcmds/image line 215.
[FAIL] Not all compute nodes are installed with the same os image.

root@c910f03c05k05:/opt/xcat/bin# 


```
1. code will return  uninitialized value $node_image_table if no node info
2.wrong message

**Comment by junxiawang** [comment]
if no diskless image on the node

**Comment by gurevichmark** [comment]
I am already making changes to `image` probe based on Xiaopeng's suggestions. Will fix this under the same pull request.

**Comment by gurevichmark** [comment]
Fixed under #1406
